tools/xchain-suite/src/pages/SwapPage.tsx

```
@@ -670,6 +707,20 @@ export default function SwapPage() {
           </div>
         )}
+        {/* DCA Callout */}
+        <Link
+          to="/recurring"
+          className="block bg-white dark:bg-gray-800 rounded-lg shadow p-4 hover:bg-gray-50 dark:hover:bg-gray-750 transition-colors group"
+        >
+          <div className="flex items-center gap-3">
+            <CalendarClock className="w-5 h-5 text-blue-500 shrink-0" />
+            <div className="flex-1 min-w-0">
+              <p className="text-sm font-medium text-gray-900 dark:text-gray-100">Dollar-Cost Average</p>
+              <p className="text-xs text-gray-500 dark:text-gray-400">Set up recurring swaps to DCA into any asset automatically</p>
```

ESLint

[error] Replace `Set·up·recurring·swaps·to·DCA·into·any·asset·automatically` with `⏎················Set·up·recurring·swaps·to·DCA·into·any·asset·automatically⏎··············`

(prettier/prettier)